en/docs/learn/consent-manager.md

```diff
@@ -65,31 +65,61 @@ Follow [Configuring users and roles](../install-and-setup/configuring-users-and-
 
 9. Go to **Portal Configurations > Subscriptions > Select Business Plan as 'Unlimited'** using the left menu pane.
 
-10. Go to **Develop > API Configurations > Runtime** using the left menu pane.   
+10. Add a custom policy. Follow the instructions given below according to the API Manager version you are using:
 
-     ![select_runtime](../assets/img/get-started/quick-start-guide/select-runtime.png)
+    ??? note "Click here to see how to add a custom policy if you are using API Manager 4.0.0..."
     
-11. Click the edit button under **Request > Message Mediation**.
-
-     ![message_mediation](../assets/img/get-started/quick-start-guide/edit-message-mediation.png)
-   
-12. Now, select the **Custom Policy** option. 
-
-13. Download the `scp-insequence.xml` file available <a href="../../assets/attachments/scp-insequence.xml" download> here </a> and use it as the Mediation Policy. 
-
-14. Upload the `scp-insequence.xml` file and click **SELECT**.
-
-15. Scroll down and click **SAVE**. 
+         1. Go to **Develop > API Configurations > Runtime** using the left menu pane.![select_runtime](../assets/img/get-started/quick-start-guide/select-runtime.png)
+            
+         2. Click the edit button under **Request > Message Mediation**.![message_mediation](../assets/img/try-out/consent-manager/message_mediation.png)
+           
+         3. Now, select the **Custom Policy** option. 
+         
+         4. Download the `scp-insequence.xml` file available <a href="../../assets/attachments/scp-insequence.xml" download> here </a> and use it as the Mediation Policy. 
+         
+         5. Upload the `scp-insequence.xml` file and click **SELECT**.
+         
+         6. Scroll down and click **SAVE**. 
+    
+    ??? note "Click here to see how to add a custom policy if you are using API Manager 4.1.0..."
+    
+         1. Go to **Develop -> API Configurations -> Policies** in the left menu pane.<br><br>
+         <div style="width:40%">
+         ![select_policies](../assets/img/get-started/quick-start-guide/select-policies.png)
+         </div>
+    
+         2. On the **Policy List** card, click on **Add New Policy**.
+    
+         3. Fill in the **Create New Policy**.
+    
+         4. Download the `scp-insequence.xml` file available <a href="../../assets/attachments/scp-insequence.xml" download> here </a> and use it as the Mediation Policy. 
```

Comment:
What is this ```scp-insequence.xml``` file? Shouldn't this be the in-sequence file related to the Berlin swagger?